src/views/splash/hoc/top-banner.jsx

```diff
@@ -0,0 +1,108 @@
+var FormattedMessage = require('react-intl').FormattedMessage;
+var injectIntl = require('react-intl').injectIntl;
+var React = require('react');
+
+var FlexRow = require('../../../components/flex-row/flex-row.jsx');
+var TitleBanner = require('../../../components/title-banner/title-banner.jsx');
+var TTTModal = require('../../../components/modal/ttt/modal.jsx');
+
+require('../../../components/forms/button.scss');
+require('./top-banner.scss');
+
+var nameTile = {
+    title: 'ttt.AnimateYourNameTitle',
+    description: 'ttt.AnimateYourNameDescription',
+    thumbUrl: '/images/ttt/animate-your-name.jpg',
+    bannerUrl: '/images/ttt/animate-your-name-banner.jpg',
+    tutorialLoc: '/projects/editor/?tip_bar=name',
+    activityLoc: 'cards.nameCardsLink',
+    guideLoc: 'guides.NameGuideLink'
+};
+
+var TopBanner = injectIntl(React.createClass({
+    type: 'TopBanner',
+    getInitialState: function () {
+        // use translated tile
+        var formatMessage = this.props.intl.formatMessage;
+        var translatedTile = {};
+        translatedTile = {
+            title: formatMessage({id: nameTile.title}),
+            description: formatMessage({id: nameTile.description}),
+            tutorialLoc: nameTile.tutorialLoc,
+            activityLoc: formatMessage({id: nameTile.activityLoc}),
+            guideLoc: formatMessage({id: nameTile.guideLoc}),
+            thumbUrl: nameTile.thumbUrl,
+            bannerUrl: nameTile.bannerUrl
+        };
+        return {currentTile: translatedTile, TTTModalOpen: false};
+    },
+    showTTTModal: function () {
+        this.setState({TTTModalOpen: true});
+    },
+    hideTTTModal: function () {
+        this.setState({TTTModalOpen: false});
+    },
+    render: function () {
+        return (
+            <TitleBanner className="mod-splash-top">
+                <FlexRow className="banner-top inner">
+                    <FlexRow className="top-animation">
+                        <img
+                            src="/images/hoc/s.png"
+                            alt="C"
```

Comment:
I think we can give null `alt`s for these, if we're using them mostly for aria standards: https://www.w3.org/WAI/tutorials/images/decorative/